Makefile

```diff
@@ -11,7 +13,7 @@ clean:
 	rm -rf build/html
 
 html: builddir
-	sphinx-build -b html $(SPHINXOPTS) . ./build/html
+	sphinx-build -j $(NUMJOBS) -b html $(SPHINXOPTS) . ./build/html
```

Comment:
It looks like as of sphinx 1.7 you can just pass `-j auto` and it'll figure it out for you: https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-j

Reply:
Ah, thanks. I did use `NUMJOBS=4 make` or so a few times locally to leave some cycles for the rest of the system, but can achieve that in other ways, too. Let me squelch it with a separate commit and if we want to have it back can revert it.

Reply:
In that case, maybe do `NUMJOBS ?= "auto"`. Then it could use the variable if you've set it.